ui/views/Authentication/ForgotPasswordView.tsx

```diff
@@ -0,0 +1,137 @@
+    let errors: FormikErrors<ForgotPasswordValues> = {};
+
+    if (values.email) {
+      if (!verifyEmailAddress(values.email)) {
+        errors['email'] = 'Please provide a valid email address.';
+      }
+    }
+
+    return errors;
+  }
+
+  function submitForgotPassword(values: ForgotPasswordValues, helpers: FormikHelpers<ForgotPasswordValues>): Promise<void> {
+    helpers.setSubmitting(true);
+
+    return sendForgotPassword(values.email, verification);
+  }
+
+  return (
+    <Fragment>
+      <div className="absolute top-2.5 left-2.5">
+        <Button
+          className="pr-3"
+          component={ RouterLink }
+          to="/login"
+        >
+          <ArrowBack className="mr-1"/>
+          Back To Login
+        </Button>
+      </div>
+      <Formik
+        initialValues={ initialValues }
+        validate={ validateInput }
+        onSubmit={ submitForgotPassword }
+      >
+        { ({
+             values,
+             errors,
+             touched,
+             handleChange,
+             handleBlur,
+             handleSubmit,
+             isSubmitting,
+             submitForm,
+           }) => (
+          <form onSubmit={ handleSubmit } className="h-full overflow-y-auto pb-20">
+            <div className="flex items-center justify-center w-full h-full max-h-full">
+              <div className="w-full p-10 xl:w-3/12 lg:w-5/12 md:w-2/3 sm:w-10/12 max-w-screen-sm sm:p-0">
+                <div className="flex justify-center w-full mb-5">
+                  <img src={ Logo } className="w-1/3"/>
+                </div>
+                <div className="w-full">
+                  <div className="w-full pb-2.5">
+                    <p className="text-center">
+                      In order to reset your forgotten password, we will send you an email with a link.
+                    </p>
+                    <p className="text-center">
+                      Please enter the email address for your login below.
+                    </p>
+                  </div>
+                  <div className="w-full pb-2.5">
+                    <TextField
+                      autoComplete="username"
+                      autoFocus
+                      className="w-full"
+                      disabled={ isSubmitting }
+                      error={ touched.email && !!errors.email }
+                      helperText={ (touched.email && errors.email) ? errors.email : null }
+                      id="forgot-email"
+                      label="Email"
+                      name="email"
+                      onBlur={ handleBlur }
+                      onChange={ handleChange }
+                      value={ values.email }
+                      variant="outlined"
+                    />
+                  </div>
+                  <CaptchaMaybe
+                    show
+                    loading={ isSubmitting }
+                    onVerify={ setVerification }
+                  />
```

Comment:
This will show whenever the `ReCAPTCHA` key is present, but not necessarily when we require captcha for this action. Have "show" based on the config.